distribution/ddf-common/src/main/resources/bin/ddf

```diff
@@ -103,7 +92,15 @@ stop_solr() {
 }
 
 print_messages() {
-    echo "DDF home is $DDF_HOME"
+    echo "Home direcotry is $HOME_DIR"
+    echo "Solr URL is $SOLR_URL"
+    local VALUE
+    if is_managing_solr; then
+      VALUE="is"
+    else
+      VALUE="is NOT"
+    fi
+    echo "The application $VALUE managing the starting and stopping of Solr"
```

Comment:
Do we need the message in both cases? If we are managing the starting and stopping of solr, we may not need to say anything more, but if not we could provide a short blurb telling them they are responsible for it (with caveats as needed).